packages/jupyter-ai-magics/jupyter_ai_magics/embedding_providers.py

```diff
@@ -66,6 +67,25 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs, **model_kwargs)
 
 
+class OllamaEmbeddingsProvider(BaseEmbeddingsProvider, OllamaEmbeddings):
+    id = "ollama"
+    name = "Ollama"
+    models = [
+        "gemma",
+        "gemma2",
+        "llama2",
+        "llama3",
+        "llama3:70b",
+        "phi3",
+        "mistral",
+        "tinyllama",
+        "qwen2",
+        "qwen2:7b",
+        "qwen2:72b",
+    ]
```

Comment:
I agree with @3coins; the list of embedding models doesn't look correct. Here is the list of embedding models I found on https://ollama.com/library:

```suggestion
    # source: https://ollama.com/library
    models = [
        "nomic-embed-text",
        "mxbai-embed-large",
        "all-minilm",
        "snowflake-arctic-embed",
    ]
```

Reply:
To obtain this list, "Ctrl + F" the page and search for "embedding".